src/libraries/System.Text.RegularExpressions/tests/UnitTests/System.Text.RegularExpressions.UnitTests.csproj

```diff
@@ -8,11 +8,14 @@
     <DebuggerSupport Condition="'$(DebuggerSupport)' == '' and '$(TargetOS)' == 'Browser'">true</DebuggerSupport>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>$(DefineConstants);DEBUG</DefineConstants> <!-- always define debug, even in release builds -->
```

Comment:
I'm not sure I follow why we are doing this. Which tests depend on it?

Reply:
The reduction tests call ToString on the RegexNodes to get a string representation to compare... but that ToString implementation is only compiled into debug builds.

Reply:
I see, makes sense. Might be useful to add a small comment above the actual call of ToString in case somebody in the future edits the test .csproj and starts getting test failures which are harder to diagnose.

Reply:
Done